src/views/ModelPlan/ReadOnly/index.tsx

```diff
@@ -232,13 +312,48 @@ const ReadOnly = () => {
           </div>
         </GridContainer>
       </SectionWrapper>
+
+      <MobileNav subComponents={subComponents} subinfo={subinfo} />
+
       <SectionWrapper className="model-plan-alert-wrapper">
         {status !== ModelStatus.CLEARED && status !== ModelStatus.ANNOUNCED && (
           <Alert type="warning" className="margin-bottom-5 desktop:margin-y-3">
             {h('alert')}
           </Alert>
         )}
       </SectionWrapper>
+
+      <SectionWrapper className="model-plan__body-content">
+        <GridContainer>
+          <Grid row gap>
+            {!isMobile && (
+              <Grid desktop={{ col: 3 }} className="padding-right-4 sticky-nav">
+                <SideNav subComponents={subComponents} />
+              </Grid>
+            )}
+
+            <Grid desktop={{ col: 9 }}>
+              <div id={subinfo ?? ''}>
```

Comment:
I think this id should be more descriptive, as I believe this is probably already used elsewhere.  Some type of prefix/suffix should work.